src/javascript/app/pages/user/metatrader/metatrader.ui.js

```diff
@@ -167,7 +168,9 @@ const MetaTraderUI = (() => {
     const setCurrentAccount = (acc_type) => {
         if (Client.get('mt5_account') && Client.get('mt5_account') !== acc_type) return;
 
-        displayAccountDescription(acc_type);
+        if (current_action_ui !== 'new_account') {
+            displayAccountDescription(acc_type);
+        }
 
         if (accounts_info[acc_type].info) {
```

Comment:
I think you can move line 172 here, this condition means that action is not new account, because accounts info is already updated with the info. please test it to confirm, though.

Reply:
![image](https://user-images.githubusercontent.com/5536948/42211890-368f13fe-7ee7-11e8-8cd1-6a341f76f5ff.png)
this check still passes on `new_account` page

Reply:
hm, I'm just wondering if there's a better way to know the action, other than using a global variable.

Reply:
another way might be to find DOM element with `act_` and selected class, like
```
if ($('.act_new_account.selected'))
```
but I think it is much worse 😄

Reply:
i don't think there is some other way of knowing what current action is, since we just update DOM on button click.
[Bind callback to all act_ buttons](https://github.com/ruskakimov/binary-static/blob/344c3b5fedd305d55b81b193e5bef63e3df253a2/src/javascript/app/pages/user/metatrader/metatrader.ui.js#L41)
[Parse action name from class name](https://github.com/ruskakimov/binary-static/blob/344c3b5fedd305d55b81b193e5bef63e3df253a2/src/javascript/app/pages/user/metatrader/metatrader.ui.js#L225)